addons/account/models/account_tax.py

```diff
@@ -1536,7 +1536,8 @@ def grouping_key_generator(base_line, tax_values):
         for tax_group_vals in tax_group_vals_list:
             tax_group = tax_group_vals['tax_group']
 
-            subtotal_title = tax_group.preceding_subtotal or _("Untaxed Amount")
+            subtotal_title = tax_group.preceding_subtotal \
+                or self.env['account.move']._fields['amount_untaxed'].get_description(self.env)['string']
```

Comment:
Interesting solution 😄 I'm not totally sure about the clarity for the user (about where to find this term to have the translation on the document), but it will indeed allow changing it.

They can always use the `preceding_subtotal` field on tax groups for this purpose. It's not translatable in version `17.0`, but it is starting from version `18.0` 😉

Reply:
It is indeed not very clear either. It's a hidden field by default on a view that is only available in debug mode. So yeah, understandable 😅